qiskit/test/mock/backends/yorktown/conf_yorktown.json

```diff
@@ -1 +1 @@
1)}(t)+U_{1}^{(0,2)}(t))\\sigma_{0}^{X} + \\Omega_{d,1}(U_{2}^{(1,0)}(t)+U_{3}^{(1,2)}(t))\\sigma_{1}^{X} \\\\ & + \\Omega_{d,2}(U_{6}^{(2,3)}(t)+U_{5}^{(2,1)}(t)+U_{7}^{(2,4)}(t)+U_{4}^{(2,0)}(t))\\sigma_{2}^{X} + \\Omega_{d,3}(U_{8}^{(3,2)}(t)+U_{9}^{(3,4)}(t))\\sigma_{3}^{X} \\\\ & + \\Omega_{d,4}(U_{11}^{(4,3)}(t)+U_{10}^{(4,2)}(t))\\sigma_{4}^{X} \\\\ \\end{align}", "h_str": ["_SUM[i,0,4,wq{i}/2*(I{i}-Z{i})]", "_SUM[i,0,4,delta{i}/2*O{i}*O{i}]", "_SUM[i,0,4,-delta{i}/2*O{i}]", "_SUM[i,0,4,omegad{i}*X{i}||D{i}]", "jq0q1*Sp0*Sm1", "jq0q1*Sm0*Sp1", "jq1q2*Sp1*Sm2", "jq1q2*Sm1*Sp2", "jq2q3*Sp2*Sm3", "jq2q3*Sm2*Sp3", "jq2q4*Sp2*Sm4", "jq2q4*Sm2*Sp4", "jq3q4*Sp3*Sm4", "jq3q4*Sm3*Sp4", "jq0q2*Sp0*Sm2", "jq0q2*Sm0*Sp2", "omegad1*X0||U0", "omegad2*X0||U1", "omegad0*X1||U2", "omegad2*X1||U3", "omegad3*X2||U6", "omegad1*X2||U5", "omegad4*X2||U7", "omegad0*X2||U4", "omegad2*X3||U8", "omegad4*X3||U9", "omegad3*X4||U11", "omegad2*X4||U10"], "osc": {}, "qub": {"0": 3, "1": 3, "2": 3, "3": 3, "4": 3}, "vars": {"delta0": -2.078515989791283, "delta1": -2.076140198460304, "delta2": -2.3632544243955147, "delta3": -2.071793501418874, "delta4": -2.092928090491978, "jq0q1": 0.011968734726718661, "jq0q2": 0.01143731530238952, "jq1q2": 0.0077637124867075335, "jq2q3": 0.011434086611531646, "jq2q4": 0.011382837107272241, "jq3q4": 0.012622615872488169, "omegad0": 0.31227678627828565, "omegad1": 0.3287329454702026, "omegad2": 0.2641052707637787, "omegad3": 0.24124632745223779, "omegad4": 0.42046738820058555, "wq0": 33.18944334684542, "wq1": 32.97119189144383, "wq2": 31.62559984414466, "wq3": 33.25055909747699, "wq4": 31.908861733581208}}, "meas_kernels": ["hw_boxcar"], "meas_levels": [1, 2], "meas_lo_range": [[6.030433052, 7.030433052], [5.981651108, 6.981651108], [5.93654928, 6.93654928], [6.078886966, 7.078886966], [6.030066921, 7.030066921]], "meas_map": [[0, 1, 2, 3, 4]], "multi_meas_enabled": true, "n_uchannels": 12, "parametric_pulses": ["gaussian", "gaussian_square", "drag", "constant"], "quantum_volume": 8, "qubit_channel_mapping": [["u4", "u0", "m0", "d0", "u1", "u2"], ["d1", "u0", "u2", "u3", "u5", "m1"], ["d2", "u4", "u7", "u8", "u10", "u1", "m2", "u3", "u5", "u6"], ["u8", "u11", "u9", "m3", "d3", "u6"], ["u7", "u11", "m4", "u9", "d4", "u10"]], "qubit_lo_range": [[4.782263967118866, 5.782263967118866], [4.747528169154703, 5.747528169154703], [4.5333705434418975, 5.533370543441897], [4.791990840932652, 5.791990840932652], [4.578453073335274, 5.578453073335273]], "rep_times": [0.001], "u_channel_lo": [[{"q": 1, "scale": [1.0, 0.0]}], [{"q": 2, "scale": [1.0, 0.0]}], [{"q": 0, "scale": [1.0, 0.0]}], [{"q": 2, "scale": [1.0, 0.0]}], [{"q": 0, "scale": [1.0, 0.0]}], [{"q": 1, "scale": [1.0, 0.0]}], [{"q": 3, "scale": [1.0, 0.0]}], [{"q": 4, "scale": [1.0, 0.0]}], [{"q": 2, "scale": [1.0, 0.0]}], [{"q": 4, "scale": [1.0, 0.0]}], [{"q": 2, "scale": [1.0, 0.0]}], [{"q": 3, "scale": [1.0, 0.0]}]], "uchannels_enabled": true, "url": "None", "allow_object_storage": true}
```

Comment:
Only change here was to remove `e+18` from `qubit_lo_range/meas_lo_range` (and removal of the new line at the end).

Reply:
These should not be updated manually. These are automatically generated. See - https://github.com/Qiskit/qiskit-terra/blob/master/tools/update_fake_backends.py to find out how to regenerate mock configs.

Reply:
Yes thx--removed the changes